app/controllers/dashboards_controller.rb

```diff
@@ -1,14 +1,18 @@
 # Basically just search and the home view
 class DashboardsController < ApplicationController
+  include LinesHelper
+
   before_action :pick_line_if_not_set, only: [:index, :search]
 
   def index
-    @urgent_patients = Patient.urgent_patients
+    @urgent_patients = Patient.urgent_patients(current_line)
     @expenditures = Patient.pledged_status_summary
   end
 
   def search
```

Comment:
Keep an eye out for merge conflicts with the issue touching search & call lines (through the patient id's) #727